tests/conftest.py

```diff
@@ -92,6 +93,18 @@ def file_store(tmp_path: Path) -> FileStore:
     return FileStore(path=tmp_path)
 
 
+@pytest.fixture()
+def file_store_create_directories(tmp_path: Path) -> FileStore:
+    shutil.rmtree(tmp_path, ignore_errors=True)  # in case the path was already created by different tests - we clean it
+    return FileStore(path=tmp_path.joinpath("subdir"), create_directories=True)
```

Comment:
I don't think this is the correct way to do this. Instead, you should create a path under `tmp_path` that has at least to levels, e.g. `tmp_path / "subdir_1" / "subdir_2". That should have the same effect, without you having to mess with pytest's `tmp_path`